src/modules/commander/PreflightCheck.cpp

```diff
@@ -272,7 +272,7 @@ static bool accelerometerCheck(orb_advert_t *mavlink_log_pub, unsigned instance,
 
 	if (ret != OK) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCEL #%u TEST FAILED: %d", instance, ret);
+			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCELEROMETER #%u TEST FAILED: %d.", instance, ret);
```

Comment:
Capitalization and wording should be updated for consistency with this PR: https://github.com/PX4/Firmware/pull/10586
Suggested new text: "Preflight Fail: accelerometer #%u test fails: %d."